bin/kano-sync

```diff
@@ -67,58 +69,77 @@ def do_login():
     login_success, value = login_using_token()
     if not login_success:
         logger.debug('token login not successful msg: {}'.format(value))
-        if args.silent:
+        if silent:
             logger.debug('exiting because of --silent')
```

Comment:
This statement is confusing - it sounds like the program is exiting just because you specified `--silent`. Maybe something less confusing like `Exiting silently after failure`?